ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1277 @@
+        "        feature_selector=feature_selector))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oYk83Zt9xF8m"
+      },
+      "source": [
+        "### Read multiple entities per request\n",
+        "\n",
+        "To read feature values from multiple entities, use the\n",
+        "StreamingReadFeatureValues API, which is almost identical to the previous\n",
+        "ReadFeatureValues API. Note that fetching only a small number of entities is recomended when using this API due to its latency-sensitive nature."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "BIJFcIIHULOd"
+      },
+      "source": [
+        "# Read the same set of features as above, but for multiple entities.\n",
+        "response_stream = data_client.streaming_read_feature_values(\n",
+        "    featurestore_online_service_pb2.StreamingReadFeatureValuesRequest(\n",
+        "        entity_type=admin_client.entity_type_path(PROJECT_ID, LOCATION,\n",
+        "                                                  FEATURESTORE_ID, \"users\"),\n",
+        "        entity_ids=[\"alice\", \"bob\"],\n",
+        "        feature_selector=feature_selector))"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "NFrVLiHyUj2l"
+      },
+      "source": [
+        "# Iterate and process response. Note the first one is always the header only.\n",
+        "for response in response_stream:\n",
+        "  print(response)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Sds42j8ZsCzS"
+      },
+      "source": [
+        "Now that you have learned how to featch imported feature values for online serving, the next step is learning how to use imported feature values for offline use cases."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "WpvhPAYxD-Ml"
+      },
+      "source": [
+        "## Batch Serving\n",
+        "\n",
+        "Batch Serving is used to fetch a large batch of feature values for high-throughput, typically for training a model or batch prediction. In this section, you will learn how to prepare for training examples by calling the BatchReadFeatureValues API."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tPzAGvqJHh3B"
+      },
+      "source": [
+        "### Use case\n",
+        "\n",
+        "**The task** is to prepare for a training dataset to train a model, which predicts if a given user will watch a given movie. To achieve this, you need 2 sets of input:\n",
+        "\n",
+        "*   Features: you already imported into the featurestore.\n",
+        "*   Labels: the groud-truth data recorded that user X has watched movie Y.\n",
```

Comment:
> typo: groud--> ground
fixed
>in that more generally you might be including not only the labels but other info too, such as (for this use case) things like referrer URL, how long spent watching, etc...
Are you suggesting that the prediction output should be more than just a yes/no label? Ideally, yes, but it's not supported yet. Actually even if the boolean label is not explicitly supported yet. Please refer to the tables down below fore more explaination.
> It would be helpful to add just a sentence on that 
It was mentioned in the env setup section, when the DESTINATION_TABLE_URI is defined. I agree that it might be too far away from where it's used but it is recommended by the template that we put all const def together in that section.
Anyways, added one more sentence as a reminder.